### PR TITLE
TotalSupporters Group Values

### PR DESCRIPTION
**Pull request by ellsclytn**
- Added group value filters
- Removed StartAt & EndAt filters

### CODE REVIEW

File: src/api/pages.js [hunk 1]
```diff
@@ -5,6 +5,11 @@ var routes    = require('./routes');
 var getJSONP  = require('../lib/getJSONP');
 var campaigns = require('./campaigns');
 
+function customJoin(ids, joinString) {
```
Comment (davidherse) [inline src/api/pages.js]: Is this function used in two places now? Might be a good idea to put it in libs as a utility function so it's not duplicated.

File: src/api/pages.js [hunk 2]
```diff
@@ -39,8 +44,13 @@ module.exports = {
   },
 
   search: function(params, callback) {
+    params.charityUid = params.charityUid ? customJoin(params.charityUid, '&charity_id[]=') : '';
+    params.campaignUid = params.campaignUid ? customJoin(params.campaignUid, '&campaign_id[]=') : '';
+    params.groupValue = params.groupValue ? customJoin(params.groupValue, '&group_value[]=') : '';
     params = _.merge({ page: 1, pageSize: 10 }, params);
+
```
Comment (davidherse) [inline src/api/pages.js]: Could delete this space

File: package.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "author": "EverydayHero <edh-dev@everydayhero.com>",
   "name": "edh-widgets",
-  "version": "0.9.1",
+  "version": "1.0.0",
```
Comment (bradparker) [inline package.json]: :tada:

File: src/lib/paramJoin.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+"use strict";
```
Comment (bradparker) [inline src/lib/paramJoin.js]: This sort of thing smells strongly of "not our concern"

Comment (bradparker) [inline src/lib/paramJoin.js]: https://github.com/ljharb/qs 16kb! woot

**Pull request merged**